src/CTA.Rules.Actions/ClassActions.cs

```diff
@@ -152,5 +160,33 @@ public class ClassActions
             };
             return RenameClass;
         }
+        public Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> GetReplaceMethodModifiersAction(string methodName, string modifiers)
+        {
+            Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> ReplaceMethodModifiers = (SyntaxGenerator syntaxGenerator, ClassDeclarationSyntax node) =>
+            {
+                var allMembers = node.Members.ToList();
+                var allMethods = allMembers.OfType<MethodDeclarationSyntax>();
+                if(allMethods.Count() > 0)
+                {
+                    var replaceMethod = allMethods.Where(m => m.Identifier.ToString() == methodName).FirstOrDefault();
+                    if(replaceMethod != null )
+                    {
+                        string[] modifierList = modifiers.Split(new char[] { ' ', ','});
+                        var verifiedModifiers = modifierList.Where(m => CTA.Rules.Config.Constants.supportedMethodModifiers.Contains(m.ToLower()));
+                        if(modifierList.Count() == verifiedModifiers.Count())
+                        {
+                            string supportedModifier = String.Join(" ", verifiedModifiers);
+                            SyntaxTokenList tokenList = new SyntaxTokenList(SyntaxFactory.ParseTokens(supportedModifier));
+                            var newMethod = replaceMethod.WithModifiers(tokenList);
```

Comment:
If above suggestion is taken, we can probably replace `supportedModifier` with `modifierList`.